src/view/concept-card.inc

```diff
@@ -1,191 +1,76 @@
-                  {% endif %}
-                  {% endif %}
-                  {% if propval.label.lang and (propval.label.lang != request.contentLang or explicit_langcodes) %}<span class="versal"> ({{ propval.label(request.contentLang).lang }})</span>{% endif %}
-                  {% if propval.SubMembers %}<div class="subvalue"> {# if property is a group concept that has sub properties #}
-                    {% for sub_member in propval.SubMembers %}
-                      <a class="propertyvalue" href="{{ sub_member.uri | link_url(propval.vocab,request.lang) }}">{{ sub_member.label(request.contentLang) }}</a>
-                      {% if sub_member.lang and (sub_member.lang != request.lang or explicit_langcodes) %}<span class="propertyvalue"> ({{ sub_member.lang }})</span>{% endif %}
-                      <br />
-                    {% endfor %}
-                    </div>
-                  {% endif %}
-                {% endif %}
-              {% elseif property.type == 'rdf:type' %}<p>{{ propval.label }}</p>
-              {% else %} {# Literals (no URI), eg. alternative labels as properties #}
-                  {% if propval.lang == request.contentLang or propval.lang == null or not request.contentLang and propval.lang == request.lang %}
-                    {% if propval.hasXlProperties %}
-                    <span class="reified-property-value xl-label tooltip-html">
-                      <img alt="Information" src="resource/pics/about.png">
-                      <div class="reified-tooltip tooltip-html-content">
-                      {% for key, val in propval.xlLabel.properties %}
-                        <p><span class="tooltip-prop">{{ key }}</span>:
-                          <span class="versal">{{ val }}</span>
-                        </p>
-                      {% endfor %}
-                      </div>
-                    </span>
-                    {% endif %}
-                    <span{% if property.type == 'skos:altLabel' %} class="replaced"{% endif %}>
-                      {%- if propval.containsHtml %}{{ propval.label|raw }}{% else %}{{ propval.label }}{% endif %}
-                      {%- if propval.lang and (request.contentLang and propval.lang != request.contentLang or explicit_langcodes) %} ({{ propval.lang }}){% endif %}
-                      {%- if propval.datatype %} ({{ propval.datatype }}){% endif -%}
-                    </span>
-                {% endif %}
-              {% endif %}
-              </td>
-            </tr>
+
+      {% for property in concept.properties %}
+      <div class="row property prop-{{property.ID}}">
+        <div class="col-sm-4 px-0 property-label"><h2>{{ property.label}}</h2></div>
+        <div class="col-sm-8 align-self-center property-value">
+          <ul class="align-bottom">
+          {% for propval in property.values %}
+            {% if propval.uri %} {# resources with URI #}
```

Comment:
This could have a clause for handling rdfs:type-property so that the property value would have only the textuar representatio of the class without a link to the rdf-representation of the class.

Reply:
Fixed